dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -875,4 +875,18 @@
     public static final String METRICS_PROTOCOL = "metrics.protocol";
 
 
+    /**
+     * Serizlization ContentTypeId
+     */
+    public static final byte AVRO_SERIALIZATION_ID = 1;
```

Comment:
this is wrong, avro is 10...

Reply:
*public static final byte AVRO_SERIALIZATION_ID = 11* should be better

Reply:
> _public static final byte AVRO_SERIALIZATION_ID = 11_ should be better

What are the uses of 1 and 5 here?

Reply:
let's simply leave it as is. I believe the original intention is: 1, 2, 4, 8, 16...

Reply:
I have modified it.